flink-cdc-cli/src/test/java/org/apache/flink/cdc/cli/CliFrontendTest.java

```diff
@@ -145,6 +146,31 @@ void testPipelineExecuting() throws Exception {
         assertThat(executionInfo.getDescription()).isEqualTo("fake-description");
     }
 
+    @Test
+    void testPipelineExecutingWithFlinkConfig() throws Exception {
```

Comment:
Please also add tests for malformed arguments (like when `=` is missing, or `=` is part of value, etc.)